src/cmd/sign.js

```diff
@@ -1,16 +1,18 @@
 /* @flow */
 import path from 'path';
 
-import {fs} from 'mz';
-import {signAddon as defaultAddonSigner} from 'sign-addon';
+import { fs } from 'mz';
+import { signAddon as defaultAddonSigner } from 'sign-addon';
 
+import { signAddon as defaultSubmitAddonSigner } from '../util/submit-addon.js';
+import type { SignResult } from '../util/submit-addon.js';
 import defaultBuilder from './build.js';
-import getValidatedManifest, {getManifestId} from '../util/manifest.js';
-import {withTempDir} from '../util/temp-dir.js';
-import {isErrorWithCode, UsageError, WebExtError} from '../errors.js';
-import {prepareArtifactsDir} from '../util/artifacts.js';
-import {createLogger} from '../util/logger.js';
-import type {ExtensionManifest} from '../util/manifest.js';
+import getValidatedManifest, { getManifestId } from '../util/manifest.js';
+import { withTempDir } from '../util/temp-dir.js';
+import { isErrorWithCode, UsageError, WebExtError } from '../errors.js';
+import { prepareArtifactsDir } from '../util/artifacts.js';
+import { createLogger } from '../util/logger.js';
+import type { ExtensionManifest } from '../util/manifest.js';
```

Comment:
would you mind to tweak your editor config to prevent it from automatically reformat the code to add spaces around the curly brackets? it would help to avoid reformatting changes to slip into the PR.

Reply:
it's not even my editor - I think I manually ran prettier at one point while I was still trying to get the editor integration working and then forgot to revert the changes.  (I do prefer the with-spaces formatting though!).  I'll get rid of it.